packages/api/src/filestorage/folder/services/googledrive/index.ts

```
@@ -136,7 +158,675 @@
+    );
+  }
+
+  private async resolveParentId(
+    parentId: string,
+    idMap: Map<string, string>,
+    driveIds: string[],
+    connectionId: string,
+    cache: Map<string, string | null>,
+  ): Promise<string | null> {
+    if (idMap.has(parentId)) {
+      return idMap.get(parentId)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)